iced_layershell/src/multi_window.rs

```diff
@@ -524,6 +524,7 @@ async fn run_instance<A, E, C>(
         };
     }
     let mut window_manager: WindowManager<A, C> = WindowManager::new();
+    let mut sizes: HashMap<iced_core::window::Id, iced_core::Size<u32>> = HashMap::new();
```

Comment:
I think here should create a new struct, to store the information of size and scale.

Reply:
I think a tuple is enough for this small use case